### PR TITLE
[codex] Add missing public docs

### PR DESCRIPTION
**Pull request by bobzhang**
## Summary

Add missing public documentation for the public APIs reported by `moon check --target all`:

- `bench` JS/native monotonic clock functions
- JS-backed `BigInt` conversion, octet, zero-check, and exponentiation APIs

## Note

For `moonbitlang/moon`: `moon --deny-warn` appears to be broken in CI, because these `missing_doc` warnings were not failing the CI run even though `moon check --target all` reported them locally.

## Validation

- `moon fmt`
- `moon check --target all`
- `moon test`
- `moon test --no-render` (passes; raw output still contains generated test-driver `missing_doc` warnings)
- `moon info` (generated `.mbti` output only had unrelated opaque-type representation churn, so it was not included)

### CODE REVIEW

**Comment by bobzhang** [comment]
cc @peter-jerry-ye it looks `moon check --deny-warn ` does not catch the missing_doc warning in the CI

File: bigint/bigint_js.mbt [hunk 1]
```diff
@@ -164,6 +164,18 @@ extern "js" fn hex2(b : Byte) -> String =
...
+/// Converts a big-endian byte sequence to a `BigInt` value with an optional
+/// sign.
+///
+/// Parameters:
+///
+/// * `octets` : A sequence of bytes representing the magnitude of the number in
+/// big-endian order.
+/// * `signum` : The sign of the resulting number. A negative value creates a
+/// negative number, zero returns zero, and a positive value creates a positive
+/// number. Defaults to 1.
```
[the remark this answers is not in the document]
Comment (bobzhang) [reply bigint/bigint_js.mbt:176]: Addressed in #3748 by adding the explicit `empty octet string` abort before the JS `BigInt("0x")` path. I also opened follow-up #3750 with regression tests for the empty-octets edge cases.
